content/en/docs/userguide/locality_loadbalance.md

```diff
@@ -0,0 +1,244 @@
+---
+draft: false
+linktitle: Locality Load Balancing
+menu:
+  docs:
+    parent: user guide
+    weight: 20
+title: Locality Load Balancing
+toc: true
+type: docs
+
+---
+
+This document introduces how to use Locality Load Balancing with Istio in the Kmesh.
+
+> The current Kemsh Locality Load Balancing is at the L4 level and only support [Locality Failover](https://istio.io/latest/docs/tasks/traffic-management/locality-load-balancing/failover/).
+
+#### What is Locality Load Balancing?
+A locality defines the geographic location of a workload instance within mesh. Locality Load Balancing in service mesh helps improve the availability and performance of services by intelligently routing traffic based on the location of the service instances.
+
+We strongly recommend that you first read https://istio.io/latest/docs/tasks/traffic-management/locality-load-balancing/ to understand what locality load balancing is.
+
+#### Supported Modes and Configuration Methods for Kmesh:
+Currently, Istio's ambient mode only supports specifying a fixed locality load balancing policy by configuring specific fields. This includes two modes: PreferClose and Local.
+
+1. PreferClose: A failover mode that uses NETWORK, REGION, ZONE, and SUBZONE as the routingPreference.
+```
+&workloadapi.LoadBalancing{
+	// Prefer endpoints in close zones, but allow spilling over to further endpoints where required.
+	RoutingPreference: []workloadapi.LoadBalancing_Scope{
+		workloadapi.LoadBalancing_NETWORK,
+		workloadapi.LoadBalancing_REGION,
+		workloadapi.LoadBalancing_ZONE,
+		workloadapi.LoadBalancing_SUBZONE,
+	},
+	Mode: workloadapi.LoadBalancing_FAILOVER,
```

Comment:
we donot need to show the underlying details. Only need to document how user can setup the service to use this feature. 

And to be more clear, list all options and the expected results